### PR TITLE
[ws-manager-mk2] Fix metric labels

### PR DESCRIPTION
**Pull request by WVerlaek**
## Description

Fix places that were still using mk1 metrics, or mk1 all-caps labels.

Remove mk1 dashboard

## Related Issue(s)

Fixes WKS-300

#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>

/hold

### CODE REVIEW

File: install/kots/manifests/kots-app.yaml [hunk 1]
```diff
@@ -33,14 +33,14 @@ spec:
     # @link https://www.gitpod.io/docs/self-hosted/latest/monitoring
     - title: Running Workspaces
       queries:
-        - query: 'sum(gitpod_ws_manager_workspace_phase_total{phase="RUNNING"}) by (type)'
+        - query: 'sum(gitpod_ws_manager_mk2_workspace_phase_total{phase="Running"}) by (type)'
```
Comment (WVerlaek) [inline install/kots/manifests/kots-app.yaml:36]: dont think this is used anymore, but just updated every reference I could find in the repo

File: operations/observability/mixins/self-hosted/dashboards/examples/overview.json [hunk 2]
```diff
@@ -112,12 +112,12 @@
       "seriesOverrides": [
         {
           "$$hashKey": "object:119",
-          "alias": "REGULAR",
+          "alias": "Regular",
```
Comment (kylos101) [inline operations/observability/mixins/self-hosted/dashboards/examples/overview.json:115]: We might be able to delete this (along with kots references), but, for a separate PR (likely for SID Team, which was the former Delivery Team).

File: operations/observability/mixins/workspace/dashboards/node-psi.json [hunk 3]
```diff
@@ -531,7 +531,7 @@
         "name": "cluster",
         "options": [],
         "query": {
-          "query": "label_values(gitpod_ws_manager_workspace_phase_total, cluster)",
+          "query": "label_values(gitpod_ws_manager_mk2_workspace_phase_total, cluster)",
```
Comment (kylos101) [inline operations/observability/mixins/workspace/dashboards/node-psi.json:534]: Nice! I was just burned by this, when checking to see why a node was struggling. :pray:

**Review by kylos101 — APPROVED** [review]
:+1:

**Review by aledbf — APPROVED** [review]

**Review by easyCZ — APPROVED** [review]

**Review by mads-hartmann — APPROVED** [review]
Awesome 🎉 

@WVerlaek to sync the alerts to Gitpod Dedicated, follow the instructions @kylos101 wrote [here](https://github.com/gitpod-io/gitpod-dedicated/tree/main/ops/telemetry/alerts#3---sync-the-alerts).

The changes to the dashboard is synced every 15 minutes ([ref](https://github.com/gitpod-io/gitpod-dedicated/blob/main/.github/workflows/grafana-cloud-dashboard-sync.yml#L6C14-L6C24)) but you can also trigger it manually if you want [here](https://github.com/gitpod-io/gitpod-dedicated/actions/workflows/grafana-cloud-dashboard-sync.yml)

**Pull request merged**